src/OpalCompiler-Core/OpalCompiler.class.st

```diff
@@ -199,12 +205,22 @@ OpalCompiler >> bindings: aDictionary [
 	self compilationContext bindings: aDictionary
 ]
 
+{ #category : #private }
+OpalCompiler >> callParsePlugins [
+	| plugins |
+
+	plugins := compilationContext astParseTransformPlugins ifEmpty: [ ^ self ].
+	plugins sort: [ :a :b | a priority > b priority ]. "priority 0 is sorted last"
+	plugins do: [ :each | ast := each transform: ast ]
+]
+
 { #category : #private }
 OpalCompiler >> callPlugins [
-	| plugins  |
+	| plugins |
```

Comment:
Now that we have some other plugins maybe we need to rename this? And keep of course the following for backwards compatibility

```smalltalk
callPlugins
   self callCompileTimePlugins
```